src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstSide;
+    private int secontSide;
+    private int height;
+
+    public IsoscelesTrapezoid(
+            String name, int firstSide, int secondSide, int height, String color) {
+        super(name, color);
+        this.firstSide = firstSide;
+        this.secontSide = secondSide;
+        this.height = height;
+    }
+
+    @Override
+    public String drawFigure() {
+        return "Figure: " + this.getName()
+                + ", area: " + this.getAria() + " sq.units, "
+                + "color: " + this.getColor();
```

Comment:
same, do we need `this` keyword here? Same in all classes

Reply:
Higher.